TheWarWithin/RogueSubtlety.lua

```diff
@@ -1475,19 +1449,31 @@ spec:RegisterAbilities( {
     symbols_of_death = {
         id = 212283,
         cast = 0,
-        charges = function() if talent.death_perception.enabled then return 1 end end,
-        cooldown = 30,
+        charges = function() if talent.death_perception.enabled then
+            return talent.death_perception.rank + 1 else return nil end
+        end,
+        cooldown = function() return 30 - ( 5 * talent.swift_death.rank ) end,
         recharge = function() if talent.death_perception.enabled then return 30 end end,
         gcd = "off",
         school = "physical",
 
+        spend = -40,
+        spendType = "energy",
+
         startsCombat = false,
 
         handler = function ()
             applyBuff( "symbols_of_death" )
-            -- applyBuff( "symbols_of_death_crit" )
 
-            if legendary.the_rotten.enabled then applyBuff( "the_rotten" ) end
+            if talent.symbolic_victory.enabled then
+                applyBuff( "symbolic_victory" )
+            end
+
+            if set_bonus.tww1 > 2 then
```

Comment:
I'm not sure if this syntax works?  Normally, it's `set_bonus.tww1_2pc > 0` -- also, as written, this looks for 3+ pieces instead of 2.

Reply:
It does work, it returns an integer of the number of pieces you have equipped from 0 to 5, I tested in-game when I was doing some hunter stuff a few weeks ago. `/dump Hekili.State.set_bonus.tww1`

I did however forget the = sign